courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2501 @@
+{
```

Comment:
<p>If we're not doing anything on the cloud, this can be deleted.</p><p> </p><p>When creating a bucket, the region should be from the list found here:</p><p><a href="https://cloud.google.com/storage/docs/locations" target="_blank">https://cloud.google.com/storage/docs/locations</a></p><p> </p><p>so using us-west1-b throws a "400" error, but using us-west1 will fix it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Cell deleted.  Thanks for the tip on bucket region.</p>